internal/resources/resources.go

```diff
@@ -148,14 +148,18 @@ func GetDragonflyResources(ctx context.Context, df *resourcesv1.Dragonfly) ([]cl
 							ImagePullPolicy: corev1.PullAlways,
 						},
 					},
-					SecurityContext: &corev1.PodSecurityContext{
-						FSGroup: &dflyUserGroup,
-					},
 				},
 			},
 		},
 	}
 
+	// Openshift requires IDs to not be set, as it injects a fixed randomized ID per namespace into all pods.
```

Comment:
This is a valid fix, but i'm not yet sure if writing to a FS works with this. Can you try running `SAVE` command on the DF instance and tell me what it says?

Thanks

Reply:
I'm not sure the correct way to run that, but I did enable snapshotting since that also writes to disk.

https://github.com/ArthurVardevanyan/HomeLab/commit/7a5ff458aeb974293d5dae7199f850f8560447f4

```
I20240311 12:20:00.002751 8 save_stages_controller.cc:321] Saving "/dragonfly/snapshots/dump-2024-03-11T12:20:00-summary.dfs" finished after 0 us
```
![image](https://github.com/dragonflydb/dragonfly-operator/assets/21159051/6dc2286d-0154-4d5d-bcfa-da294540ca33)

Reply:
That does seem to work!